make-manifest.sh

```diff
@@ -0,0 +1,269 @@
+#!/bin/bash
+
+# Usage: ./mirror.sh <image_src>
+
+# Requires manifest-tool for inspecting from gcr.
+#   GO111MODULE=off go get github.com/estesp/manifest-tool
+# Requires Docker experimental features to be enabled to run `docker manifest`
+# Can mirror existing manifests or create new manifests from template.
+# Forces os, arch, & variant for platform specific tags (modifies image).
+# Platform tags as REPO:OS-ARCH(-VARIANT).
+
+set -e
+
+PLATFORMS=${PLATFORMS:-'linux/amd64,linux/arm64,linux/arm'}
```

Comment:
This needs to be explicit I think, we don't want to default to platforms, it needs to be specified correctly.